simulation-model/simulation-model_instance.json

```diff
@@ -0,0 +1,194 @@
+            },
+            "general:contractId": {
+                "@value": "1234567890",
+                "@type": "xsd:string"
+            },
+            "general:recordingTime": {
+                "@value": "2024-06-26T09:00:00",
+                "@type": "xsd:dateTime"
+            }
+        },
+        "general:links": {
+            "@type": "general:Links",
+            "general:data": {
+                "@type": "general:Link",
+                "general:url": {
+                    "@value": "https://github.com/openMSL/sl-1-1-reflection-based-radar-object-model",
+                    "@type": "xsd:anyURI"
+                },
+                "general:type": "Asset"
+            },
+            "general:media": {
+                "@type": "general:Link",
+                "general:url": {
+                    "@value": "https://inavlid.link.com/media",
+                    "@type": "xsd:anyURI"
+                },
+                "general:type": "Video"
+            }
+        },
+        "general:bundleData": {
+            "@type": "general:BundleData",
+            "general:requiredData": {
+                "@type": "general:Link",
+                "general:url": {
+                    "@value": "did:web:registry.gaia-x.eu:OSITrace:T9rkdS19poUrbbAEk3rXMqZ3hzVezwDrrxaf",
+                    "@type": "xsd:anyURI"
+                },
+                "general:type": "Asset"
+            },
+            "general:relatedData": {
+                "@type": "general:Link",
+                "general:url": {
+                    "@value": "did:web:registry.gaia-x.eu:OSITrace:T9rkdS19poUrbbAEk3rXMqZ3hzVezwDrrxaf",
+                    "@type": "xsd:anyURI"
+                },
+                "general:type": "Asset"
+            }
+        }
+    },
+    "simulation-model:format": {
+        "@type": "simulation-model:Format",
+        "https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/simulation-model/model.implementation.format": {
+            "@value": "OSMP 1.0/OSI 3.1.2/FMI 2.0",
+            "@type": "xsd:string"
+        },
+        "https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/simulation-model/model.preconditions": {
+            "@value": "Object-list input contains accurate bounding box values",
+            "@type": "xsd:string"
+        },
+        "https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/simulation-model/model.implementation.preconditions": {
+            "@value": "Model needs Windows 10 1809 or newer with nVidia Drivers version 389.12 or newer and OpenCL 2.0 capable GPU",
+            "@type": "xsd:string"
+        },
+        "https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/simulation-model/model.integration.requirements": {
+            "@value": "http://mywebsite.de/Requirements.pdf",
+            "@type": "xsd:anyURI"
+        }
+    },
+    "simulation-model:content": {
+        "@type": "undefined",
```

Comment:
Is this intended to be undefined?

Reply:
Nope, I've created the example through the wizard and somehow this was undefined and i didn't see it as well as the usual checks. Thanks! (fixed now)